lambdas/access_counts/index.py

```diff
@@ -169,24 +169,40 @@ def sql_escape(s):
 PACKAGE_ACCESS_COUNTS = textwrap.dedent("""\
     SELECT
         eventname,
-        package_hashes.bucket AS bucket,
+        bucket,
         name,
-        CAST(histogram(date) AS JSON) AS counts
-    FROM object_access_log JOIN package_hashes
-    ON object_access_log.bucket = package_hashes.bucket AND key = concat('.quilt/packages/', hash)
+        CAST(map_agg(date, count) AS JSON) AS counts
+    FROM (
+        SELECT
+            eventname,
+            t.bucket AS bucket,
+            name,
+            date,
+            sum(count) AS count
+        FROM (
+            SELECT eventname, bucket, substr(key, 17) AS hash, date, count(*) AS count from object_access_log
+            WHERE substr(key, 1, 16) = '.quilt/packages/'
```

Comment:
Easier to maintain if you do `MANIFEST_PREFIX = '.quilt/packages/'` above and `len(MANIFEST_PREFIX)`, etc. here.